BUILD

```diff
@@ -543,7 +562,9 @@ grpc_cc_library(
         "include/grpc/census.h",
     ],
     deps = [
-        "grpc_base",
+        "gpr_base",
+        "grpc_base_c",
```

Comment:
Why do we have `grpc_base` vs. `grpc_base_c` in the first place?  It looks like the former is just the latter plus the lame_client filter.  Is there any reason why that is split out?

I'm guessing that this is a leftover from the fact that the lame_client filter was the very first piece of code we converted from C to C++.  It's probably not necessary anymore.  I suggest just moving the lame_client filter into `grpc_base_c` and renaming it to `grpc_base`.

Reply:
Happy to file a bug to do so, but I don't think it should be in this PR.

Reply:
(I think the reason is what you expect)